src/pages/TeamPage/index.js

```diff
@@ -0,0 +1,348 @@
+                  variant="contained"
+                  color="primary"
+                  className={!isActive ? classes.team_btn : classes.team_btn1}
+                  sx={
+                    !isActive
+                      ? { marginRight: 4 }
+                      : { marginRight: 0, marginBottom: 2 }
+                  }
+                >
+                  Join as a Partner
+                </Button>
+                <Button
+                  variant="outlined"
+                  color="primary"
+                  spacing={2}
+                  container
+                  className={!isActive ? classes.team_btn : classes.team_btn1}
+                >
+                  Volunteer at Meraki
+                </Button>
+              </Grid>
+            </Stack>
+          </Container>
+          <Container
+            className={
+              !isActive
+                ? classes.team_containerTopSpace
+                : `${classes.team_responsiveContainer}`
+            }
+          >
+            <Grid container style={{ justifyContent: "center" }}>
+              <Typography
+                onClick={() => {
+                  setMembers({ volunteers: false, teamMembers: true });
+                }}
+                variant="subtitle1"
+                className={
+                  !isActive
+                    ? classes.team_selector
+                    : classes.team_MobileSelector
+                }
+                style={
+                  members.teamMembers
+                    ? {
+                        fontWeight: "bold",
+                        borderBottom: "3px solid #48a145",
+                      }
+                    : {}
+                }
+              >
+                Core Team
+              </Typography>
+              <Typography
+                onClick={() => {
+                  setMembers({ teamMembers: false, volunteers: true });
+                }}
+                variant="subtitle1"
+                className={
+                  !isActive
+                    ? classes.team_selector
+                    : classes.team_MobileSelector
+                }
+                style={
+                  members.volunteers
+                    ? {
+                        fontWeight: "bold",
+                        borderBottom: "3px solid #48a145",
+                      }
+                    : {}
+                }
+              >
+                Our Supporters
+              </Typography>
+            </Grid>
+            <Container
+              className={
+                !isActive
+                  ? `${classes.team_infoCardContaier} team-info-cards-container`
+                  : `${classes.team_infoResponsiveContainer} team-info-cards-container`
+              }
+              sx={{ marginTop: 4 }}
+            >
+              <Grid container>
+                {Object.keys(shuffleObject(team)).length ? (
+                  Object.keys(shuffleObject(team)).map((item) => {
```

Comment:
No need to shuffle twice and using the above suggestion, you could do the following:
```suggestion
                {Object.keys(team).length ? (
                  shuffleArray(Object.keys(team)).map((item) => {
```